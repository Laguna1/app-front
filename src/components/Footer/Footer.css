.nav-menu {
  color: rgb(0, 255, 234);
  padding: 20px;
}
