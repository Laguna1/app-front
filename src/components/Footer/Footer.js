import React from 'react';
import './Footer.css';

const Footer = () => (

  <footer className="footer">FOOTER</footer>

);

export default Footer;
