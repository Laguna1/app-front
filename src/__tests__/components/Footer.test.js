import renderer from 'react-test-renderer';
import thunk from 'redux-thunk';
import { Provider } from 'react-redux';
import { createStore, applyMiddleware } from 'redux';
import { BrowserRouter } from 'react-router-dom';
import combinedReducers from '../../reducers';
import Footer from '../../components/Footer/Footer';

test('Footer renders correctly', () => {
  const store = createStore(combinedReducers, applyMiddleware(thunk));
  const tree = renderer
    .create(
      <Provider store={store}>
        <BrowserRouter>
          <Footer />
        </BrowserRouter>
      </Provider>,
    ).toJSON();
  expect(tree).toMatchSnapshot();
});
