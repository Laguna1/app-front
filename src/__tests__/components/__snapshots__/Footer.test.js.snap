// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<nav
  className="menu"
>
  <a
    href="/main"
    onClick={[Function]}
  >
    <div
      className="icons"
    >
      <i
        className="fa fas fa-walking"
      />
      <p>
        
      </p>
    </div>
  </a>
  <a
    href="/login"
    onClick={[Function]}
  >
    <div
      className="icons"
    >
      <i
        className="fa fas fa-sign-in"
      />
      <p>
        
      </p>
    </div>
  </a>
  <a
    href="/signup"
    onClick={[Function]}
  >
    <div
      className="icons"
    >
      <i
        className="fa fas fa-user-plus"
      />
      <p>
        
      </p>
    </div>
  </a>
</nav>
`;
